quartz/bootstrap-cli.mjs

```diff
@@ -41,6 +41,23 @@ const CommonArgv = {
     default: false,
     describe: "print out extra logging information",
   },
+  "in-directory": {
+    string: true,
+    alias: ["i"],
+    describe: "what directory to copy/create symlink from",
+  },
+  setup: {
+    string: true,
+    alias: ["s"],
+    choices: ["new", "copy", "symlink"],
+    describe: "strategy for content folder setup",
+  },
+  links: {
+    string: true,
+    alias: ["l"],
+    choices: ["absolute", "shortest", "relative"],
+    describe: "strategy to resolve links",
+  },
```

Comment:
`CommonArgv` is the shared argv that every command uses. Let's create a `CreateArgv` that spreads `CommonArgv` and adds these